### ISSUE TITLE
Add aria-current="page" to the active link

### ISSUE THREAD

**Issue by phiter** [vuejs/vue-router#3144]
### What problem does this feature solve?
Users that can see the page can have a nice experience of knowing which link is currently active just by looking at a styled active link with the `activeClass`.

But visually impaired users, those who rely on screen readers, currently have no way of knowing that the currently selected link is the active link.

To fix this, `aria-current="page"` adds an extra information to the screen reader that the currently focused link is the same as the current page, so if we add that together with `activeClass` we'll be aiding both sighted and blind users.

### What does the proposed API look like?
Basically, when we add `activeClass` also pass `aria-current="page"` to the generated anchor.

**Comment by posva** [comment]
Duplicate of https://github.com/vuejs/vue-router/issues/2116
